src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -0,0 +1,127 @@
+    <div className="AccessApprovalsTable bootstrap-4-backport">
+      <Typography style={{ padding: '12px' }} variant="headline3">
+        Status in Access Requirements
+      </Typography>
+      <Table striped borderless bordered={false}>
+        <thead className="access-approval-header">
+          <tr>
+            <th className="thead-cell remove-border">AR ID</th>
+            <th className="thead-cell remove-border">
+              Access Requirement Name
+            </th>
+            <th className="thead-cell remove-border">Submitter</th>
+            <th className="thead-cell remove-border">Status</th>
+            <th className="thead-cell remove-border button-cell">
+              Modified Date
+              <button
+                style={{ float: 'right' }}
+                onClick={() =>
+                  onSort(AccessApprovalSortField.MODIFIED_ON, isDescending)
+                }
+              >
+                <IconSvg
+                  options={{
+                    icon: 'sortTwoTone',
+                  }}
+                />
+              </button>
+            </th>
+            <th className="thead-cell remove-border button-cell">
+              Expires
+              <button
+                style={{ float: 'right' }}
+                onClick={() =>
+                  onSort(AccessApprovalSortField.EXPIRED_ON, isDescending)
+                }
+              >
+                <IconSvg
+                  options={{
+                    icon: 'sortTwoTone',
+                  }}
+                />
+              </button>
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {''}
+          {accessApprovals.map(item => {
+            let modifiedOn = moment(item.modifiedOn).format('L LT')
+            let expiredOn = moment(item.expiredOn).format('L LT')
+            return (
+              <tr key={item.id}>
+                <td className="remove-border">
+                  <a
+                    href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!AccessRequirement:AR_ID=${item.accessRequirementId}`}
+                  >
+                    {item.accessRequirementId}
+                  </a>
+                </td>
+                <td className="remove-border">{item.accessRequirementName}</td>
+                <td className="remove-border">
+                  <UserCard size={SMALL_USER_CARD} ownerId={item.submitterId} />
+                </td>
+                <td className="remove-border">
+                  {item.state[0] + item.state.slice(1).toLocaleLowerCase()}
+                </td>
+                <td className="remove-border">{modifiedOn}</td>
+                <td
+                  className={`${
+                    new Date() > new Date(expiredOn) ? 'expired' : ''
+                  } remove-border`}
+                >
+                  {expiredOn}
+                </td>
+              </tr>
+            )
+          })}
+        </tbody>
+      </Table>
+    </div>
```

Comment:
Seems we lost the 'Show More' button, which would call `fetchNextPage` returned by the infinite query hook. The button should only be shown when `hasNextPage` (also returned by the hook) is true, and should be disabled or hidden while fetching the next page.